apps/dashboard/src/components/subscribers/create-subscriber-form.tsx

```diff
@@ -88,6 +89,7 @@ export const CreateSubscriberForm = (props: CreateSubscriberFormProps) => {
     await createSubscriber({
       subscriber: { ...dirtyPayload, subscriberId: formData.subscriberId },
     });
+    navigateToSubscribersFirstPage();
```

Comment:
will be better to move this to the `onSuccess` callback of the `useCreateSubscriber` hook, else it will redirect to the first page on network error too

Reply:
in case of network error the promise will be rejected and won't go to the next line, but I'll move it to the `onSuccess`